desktop/cmp/filter/FilterChooser.js

```diff
@@ -71,7 +73,20 @@ export const [FilterChooser, filterChooser] = hoistCmp.withFactory({
 });
 
```

Comment:
I actually think we should not pass on the '...rest' but explicity pass on the arguments.

Reply:
we already have lines for 3 of the 5 that can incorporate props

Reply:
Good point, makes it tighter - will update